mob-ai-csharp/mob-ai-csharp.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Remove="lib\Newtonsoft.Json.dll" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="lib\Newtonsoft.Json.dll">
```

Comment:
Where did this file came from? Could you provide a source? I need to check the licensing, and eventually move its download to the compilation step instead of including it directly in the repo.

Reply:
Newtonsoft Json It's MIT License, you can find it here -> https://github.com/JamesNK/Newtonsoft.Json/blob/master/LICENSE.md